gdal is broken on Debian Wheezy

```
[using autodetected 6 job(s)]
make[1]: Entering directory `/home/mxe/mxe-16Sep2015'
uname -a
Linux sd-68075 3.2.0-4-amd64 #1 SMP Debian 3.2.63-2+deb7u1 x86_64 GNU/Linux
git show-branch --list --reflog=1
  [refs/heads/master@{0}] (5 hours ago) clone: from https://github.com/mxe/mxe
lsb_release -a 2>/dev/null || sw_vers 2>/dev/null || true
Distributor ID:	Debian
Description:	Debian GNU/Linux 7.8 (wheezy)
Release:	7.8
Codename:	wheezy
autoconf --version 2>/dev/null | head -1
autoconf (GNU Autoconf) 2.69
automake --version 2>/dev/null | head -1
automake (GNU automake) 1.11.6
rm -rf   '/home/mxe/mxe-16Sep2015/tmp-gdal-i686-w64-mingw32.static'
mkdir -p '/home/mxe/mxe-16Sep2015/tmp-gdal-i686-w64-mingw32.static'
cd '/home/mxe/mxe-16Sep2015/tmp-gdal-i686-w64-mingw32.static' &&  tar xzf '/home/mxe/mxe-16Sep2015/pkg/gdal-1.11.2.tar.gz'
cd '/home/mxe/mxe-16Sep2015/tmp-gdal-i686-w64-mingw32.static/gdal-1.11.2'
(cd '/home/mxe/mxe-16Sep2015/tmp-gdal-i686-w64-mingw32.static/gdal-1.11.2' && patch -p1 -u) < ./src/gdal-1-fixes.patch
patching file m4/geos.m4
patching file configure.in
Hunk #3 succeeded at 2459 (offset 12 lines).
Hunk #4 succeeded at 2765 (offset 9 lines).
Hunk #5 succeeded at 2807 (offset 9 lines).
patching file m4/geos.m4
cd '/home/mxe/mxe-16Sep2015/tmp-gdal-i686-w64-mingw32.static/gdal-1.11.2' && autoreconf -fi
libtoolize: putting auxiliary files in `.'.
libtoolize: copying file `./ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: copying file `m4/libtool.m4'
libtoolize: copying file `m4/ltoptions.m4'
libtoolize: copying file `m4/ltsugar.m4'
libtoolize: copying file `m4/ltversion.m4'
libtoolize: copying file `m4/lt~obsolete.m4'
libtoolize: Consider adding `-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
libtoolize: `AC_PROG_RANLIB' is rendered obsolete by `LT_INIT'
configure.in:78: error: possibly undefined macro: AC_LD_SHARED
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.in:106: error: possibly undefined macro: AC_HAVE_LONG_LONG
configure.in:107: error: possibly undefined macro: AC_UNIX_STDIO_64
configure.in:158: error: possibly undefined macro: AC_CHECK_FUNC_CUSTOM
configure.in:4055: error: possibly undefined macro: AC_CHECK_FW_FUNC
autoreconf: /usr/bin/autoconf failed with exit status: 1
make[1]: *** [build-only-gdal_i686-w64-mingw32.static] Error 1
make[1]: Leaving directory `/home/mxe/mxe-16Sep2015'

real	0m9.069s
user	0m7.608s
sys	0m0.232s
```

Currently `gdal` is the only broken package on Debian Wheezy.

--- a/src/gdal.mk
+++ b/src/gdal.mk
@@ -20,7 +20,7 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_BUILD
-    cd '$(1)' && autoreconf -fi
+    cd '$(1)' && autoreconf -fi -I ./m4
     # The option '--without-threads' means native win32 threading without pthread.
     cd '$(1)' && ./configure \
         $(MXE_CONFIGURE_OPTS) \

